src/jqLite.js

```diff
@@ -671,7 +671,10 @@ forEach({
       }
       return element.value;
     }
-    element.value = value;
+    // Workaround for Firefox validation #12102.
+    if (element.value != value) {
```

Comment:
Yes,  but `value` can be anything the user passes (e.g. `0`).
If `element.value === ''` and `value === 0`, I would expect the element's value to be set to `'0'`, but because `'' != 0` evaluates to false, the element's value will remain `''`, right ?